specification/dimensions/commitmentdiscounttype.md

```diff
@@ -0,0 +1,32 @@
+# Commitment Discount Type
+
+A commitment-based discount is a commitment for an amount of usage or spend throughout a specified term, in exchange for discounted unit pricing on that amount. The commitment may be based on quantities of resource units or monetary value, with various payment options and time frames.
```

Comment:
This is a very new pricing model ([EC2 Capacity Blocks](https://aws.amazon.com/ec2/capacityblocks/)) within AWS that allows a 1 to 14-day term commitment.  The only net-difference that I see (other than term length from RI's/SP's across providers) is that after a commitment is bought, an additional charge for the OS-chosen may apply.  See the [pricing table](https://aws.amazon.com/ec2/capacityblocks/pricing/) for examples.

```suggestion
A commitment-based discount is a commitment for an amount of usage or spend throughout a specified term, in exchange for discounted unit pricing on that amount. The commitment may be based on quantities of resource units or monetary value, with various payment options, time frames, and/or additional usage charges.
```